plugins/jobs_and_leads/server.js

```diff
@@ -0,0 +1,108 @@
+const dummyBusinessContactInfo = [
```

Comment:
Are these all fake? I don't want to be adding real businesses here...

Reply:
It's just dummy data from ChatGPT to match the Google Places api response

Reply:
Okay.